### ISSUE TITLE
🐛 Poor colour contrast for `disabled` state of `RichTextInput` component

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#14429]
### 🐛 Bug

The `disabled` state of the `RichTextInput` component on light mode has very poor colour contrast with dark purple on dark grey.

### 📋 Steps to Reproduce

1. Navigate to http://localhost:8000/en/admin/training-opportunities/create
2. Enter text, select the different styles of the Description field
3. Notice the purple text

### 📸 Screenshot

<img width="1419" height="605" alt="Screenshot 2025-08-22 at 14 27 26" src="https://github.com/user-attachments/assets/7eb069cf-9b48-42fb-9227-b30eeb7eb391" />

<img width="1412" height="571" alt="Screenshot 2025-08-22 at 14 27 14" src="https://github.com/user-attachments/assets/77e2ffc9-d4b0-4d8d-a1a7-7ac8248e2a2e" />

### ✅ Acceptance Criteria

- [ ] colour contrast meets WCAG 2.1 and is legible

**Comment by petertgiles** [comment]
@mnigh Can you see what it should be from Figma?

**Comment by mnigh** [comment]
> [@mnigh](https://github.com/mnigh) Can you see what it should be from Figma?

I can't find a design for that state in either of the Figma files that would be relevant:

- Component: https://www.figma.com/design/guHeIIh8dqFVCks310Wv0G/Component-library?node-id=5-13&p=f&m=dev
- Page design: https://www.figma.com/design/Qyh0TgZIlY3BStBNqKxO9t/Admin---Training-event-manager?node-id=16124-9694&m=dev

@Jerryescandon can you provide some guidance on how it should render?

**Comment by Jerryescandon** [comment]
Actually yeah that makes sense. Lets do that
